ResearchKit/Consent/ORKConsentLearnMoreViewController.h

```diff
@@ -38,6 +38,7 @@ NS_ASSUME_NONNULL_BEGIN
 @interface ORKConsentLearnMoreViewController : UIViewController
 
 - (instancetype)initWithHTMLContent:(NSString *)content;
+- (instancetype)initWithContentRequest:(NSURLRequest *)contentRequest;
```

Comment:
One use case would be to set the authentication headers.

J

Sent from my iPhone 6plus

> On May 22, 2015, at 1:52 PM, jwe-apple <notifications@github.com> wrote:
> 
> In ResearchKit/Consent/ORKConsentLearnMoreViewController.h:
> 
> > @@ -38,6 +38,7 @@ NS_ASSUME_NONNULL_BEGIN
> >  @interface ORKConsentLearnMoreViewController : UIViewController
> >  
> >  - (instancetype)initWithHTMLContent:(NSString *)content;
> > +- (instancetype)initWithContentRequest:(NSURLRequest *)contentRequest;
> Right, but would there ever be a need to specify any parameter of the request other than the URL?
> 
> —
> Reply to this email directly or view it on GitHub.
>